build/progress.js

```diff
@@ -10,7 +10,7 @@ class Progress extends ProgressBar {
     super(`${string.padEnd(24, ' ')}: ${colors.green('[')}:bar${colors.green(']')} :current/:total (:elapseds) :etas remaining`, {
       incomplete: colors.red('-'),
       width: 20,
-      total
+      total: total || 0
```

Comment:
probably not needed, i think this was an error caused because i was on top of nightwave data